generate_build_config.py

```diff
@@ -26,30 +27,41 @@
 
 # Perform the build
 - /home/ubuntu/launchpad-buildd/mount-chroot $BUILD_ID
-- /home/ubuntu/launchpad-buildd/update-debian-chroot $BUILD_ID
+- /home/ubuntu/launchpad-buildd/update-debian-chroot $BUILD_ID{}
```

Comment:
Could we make this a named placeholder? It makes reading through the template a little easier.